### ISSUE TITLE
Application could not be started, or no application associated with the specified file. ShellExecuteEx failed: File not found.

### ISSUE THREAD

**Issue by zaphida** [Bladez1992/HardLaunch#1]
On Linux with wine version wine-8.2-110-g4d0d7e49b9b, I got the above error.
Full console output:
```
kernel  ~  Downloads  HardLaunch (v1.0.9.9)  wine HardLaunch.bat
0078:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\CdaC15BA": c000000d
003c:fixme:service:scmdatabase_autostart_services Auto-start service L"CdaC15BA" failed to start: 87
00d8:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
00d8:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
00d8:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
00d8:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
                        HardLaunch                                              
                                --- Version Switcher ---                        
1.  Run Hardwar Demo - ver. D1.6                                                
2.  Run Hardwar - ver. F2.0                                                     
3.  Run Hardwar - ver. F2.1                                                     
4.  Run Hardwar - ver. U3.00b1                                                  
5.  Run Hardwar - ver. U3.00b2                                                  
6.  Run Hardwar - ver. U3.00b3                                                  
7.  Run Hardwar - ver. U3.00b4                                                  
8.  Run Hardwar - ver. U3.00b5                                                  
9.  Run Hardwar - ver. UIM.01                                                   
10. Run Hardwar - ver. UIM.02 - Lost to time...                                 
11. Run Hardwar - ver. UIM.02.01                                                
                        HardLaunch                                              
                         v1.0.7.4                                               
                      By Bladez1992                                             
     Credit to Lord Solarix for finding some of these commands!                 
--------------------------------------------------------------------------      
1.  Create A New Local World                                                    
2.  Create A New Network Game                                                   
3.  Create A New Dedicated Server                                               
4.  Load A Local World                                                          5.  Load A Network Game                                                         6.  Load A Dedicated Server                                                     
7.  Connect (Connect to a Network Game or Dedicated Server)                     
8.  Change Resolution                                                           
9.  Graphics Options                                                            
10. Audio Options                                                               
11. Network Options                                                             
12. Debug/Other/Unknown Functions                                               
                                                                                
13. Change Interface Colors                                                     
--------------------------------------------------------------------------      
Type the number of your selection: 1                                            
"What would you like your Local World to be called: "meow                       
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100                         
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100                         
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100
019c:fixme:exec:SHELL_execute flags ignored: 0x00000100
Application could not be started, or no application associated with the specifie
d file.                                                                         
ShellExecuteEx failed: File not found.

```

**Comment by Bladez1992** [comment]
It seems to me that wine is missing some critical Windows functions

0078:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\CdaC15BA": c000000d
Here is a potential fix
askubuntu.com/questions/1446664/ntoskrnlzwloaddriver-failed-to-create-driver

As for the 019c:fixme:exec error, I'm 90% sure it's wine related. Googling the error shows that it happens on other Windows software as well.

Also keep in mind I did not design HardLaunch with Linux in mind

**Comment by Bladez1992** [comment]
Fixed this issue as of v1.1.1.7 - HardLaunch still does not work on Wine, seemingly due to Wine mishandling quotes and directory commands in Windows batch files
